user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Template_Editor.md

```diff
@@ -0,0 +1,338 @@
+| ![Top](~/user-guide/images/Dimensions_Top.png) | Top | Specify a number of pixels to which the top of the layer will be fixed. This value should be lower than the height of the overall template. |
+| ![Bottom](~/user-guide/images/Dimensions_Bottom.png) | Bottom | Specify a number of pixels to which the bottom of the layer will be fixed. This value should be lower than the height of the overall template. |
+| ![Left](~/user-guide/images/Dimensions_Left.png) | Left | Specify a number of pixels to which the left of the layer will be fixed. This value should be lower than the width of the overall template. |
+| ![Right](~/user-guide/images/Dimensions_Right.png) | Right | Specify a number of pixels to which the right of the layer will be fixed. This value should be lower than the width of the overall template. |
+
+> [!NOTE]
+>
+> - In the preview, when you use the drag-and-drop functionality to move a layer, or when you resize a layer by dragging the frame edges, the dimension settings in the *Settings* pane will automatically adjust.
+> - You can change the unit to percentages by clicking *px* next to the dimension values.
+> - You can use the lock functionality to enhance the security of your design, locking one or multiple dimension settings of a layer by clicking the ![*Lock*](~/user-guide/images/Dimensions_Lock.png) button next to the chosen setting. Once locked, any attempt to override this specified dimension will be restricted, including moving or resizing that layer in the preview and resizing the component in the dashboard or low-code app edit mode (e.g. changing the size of a table column). The locked dimensions will ensure the template remains in its designated position. If you cannot lock a dimension setting because of the locking of other settings, it will appear grayed out.
+> - For some layers, the width and height is determined by the left and right position and can therefore not be modified.
+> - When resizing a layer in the preview, do not release the mouse button if one or more dimension settings are surrounded by a red frame, signaling that they no longer meet the minimum required pixel count. Releasing the mouse button in this state may result in the deletion of your layer.
+
+### Specifying layer properties
+
+Depending on the type of layer, you can specify different layer properties in the *Settings* pane.
+
+- For *Icon* layers, you can specify the following properties:
+
+  - *Show icon*: Allows you to mask the layer, rendering it temporarily invisible. If disabled, the layer is still visible in the preview with lowered opacity, but will not be visible in the template's end result.
+
+  - *Color*: Allows you to specify a custom icon color, either by specifying the color in RGB format or by using the color picker box on the right.
```

Comment:
Can be a Hex value asswel

Reply:
or even by name: https://www.w3schools.com/tags/ref_colornames.asp